### PR TITLE
Add documentation to model initializers

### PR DESCRIPTION
**Pull request by pelargir**
I was unsure about some of these types so please let me know if anything is wrong.

### CODE REVIEW

File: lib/friendly_shipping/label.rb [hunk 1]
```diff
@@ -14,6 +14,17 @@ class Label
...
+    # @param [Integer] id The label's ID
+    # @param [Integer] shipment_id The label's shipment ID
+    # @param [String] tracking_number The label's tracking number
+    # @param [String] service_code The label's service code
+    # @param [String] label_href The URL for the label
+    # @param [String] label_format The label's format
+    # @param [String] label_data The raw label data
+    # @param [Float] shipment_cost The cost of the shipment
+    # @param [Hash] data Additional data related to the label
+    # @param [FriendlyShipping::Request] original_request The HTTP request used to obtain the label
```
Comment (mamhoff) [inline lib/friendly_shipping/label.rb]: I would like to remove these two options as they are available on the `ApiResult` class.

File: lib/friendly_shipping/rate.rb [hunk 2]
```diff
@@ -13,6 +13,15 @@ class NoAmountsGiven < StandardError; end
...
                 :original_response
 
+    # @param [FriendlyShipping::ShippingMethod] shipping_method The rate's shipping method
+    # @param [Hash] amounts The amounts (as Money objects) that make up the rate
+    # @param [Integer] remote_service_id The remote service ID for the rate
+    # @param [Time] delivery_date The delivery date for the rate
+    # @param [Array] warnings Any warnings that were generated
+    # @param [Array] errors Any errors that were generated
+    # @param [Hash] data Additional data related to the rate
+    # @param [FriendlyShipping::Request] original_request The HTTP request used to obtain the rate
```
Comment (mamhoff) [inline lib/friendly_shipping/rate.rb]: These last two should IMO go as they are available on the `ApiResult` class.

File: spec/friendly_shipping/response_spec.rb [hunk 3]
```diff
@@ -1,4 +1,4 @@
-# frozen_string_literal: true
+{}# frozen_string_literal: true
```
Comment (mamhoff) [inline spec/friendly_shipping/response_spec.rb]: Typo

File: spec/friendly_shipping/rate_spec.rb [hunk 4]
```diff
@@ -10,11 +10,14 @@
...
   end
   it { is_expected.to respond_to(:shipping_method) }
+  it { is_expected.to respond_to(:amounts) }
   it { is_expected.to respond_to(:remote_service_id) }
   it { is_expected.to respond_to(:delivery_date) }
   it { is_expected.to respond_to(:warnings) }
   it { is_expected.to respond_to(:errors) }
   it { is_expected.to respond_to(:data) }
+  it { is_expected.to respond_to(:original_request) }
+  it { is_expected.to respond_to(:original_response) }
```
Comment (mamhoff) [inline spec/friendly_shipping/rate_spec.rb]: Same here, I don't think we need these any longer on `Rate`.

File: spec/friendly_shipping/response_spec.rb (hunk 3 above)
Comment (pelargir) [reply spec/friendly_shipping/response_spec.rb]: Fixed.

**Comment by pelargir** [comment]
@mamhoff good to know some of those params can be removed. I agree that this would be best done in a subsequent PR.

**Comment by pelargir** [comment]
The build has been pending for an hour now. Are we out of CI credits for the period?

**Pull request merged**